src/main/java/uk/gov/gchq/palisade/util/AbstractResourceBuilder.java

```diff
@@ -29,9 +29,9 @@
 /**
  * ResourceBuilder, taking a URI and building a Resource specific to each scheme
```

Comment:
The class was ResourceBuilder and now is AbstractResourceBuilder.  The Java Doc don't reflect this plus do not make a lot of sense.  Suggestion would be to change the documentation.  The current class JavaDoc is "ResourceBuilder, taking a URI and building a Resource specific to each scheme".  Suggestion would be something more along the lines of AbstractResourceBuilder is an abstract factory for Resource objects.  Given a URI, it will identify the appropriate implementation and use it to produce a Resource.    Not really the ticket to improve the JavaDoc so if just the class document was at least improved, so in my opinion this would be adequate.  The rest is stuff is like the use of words like scheme and cache without much clarity as to what they mean, the word resource being use when I think that Resource would be more appropriate etc.

Reply:
The class JavaDoc has been reworked, plus a few other JavaDoc updates for the methods.